composer.json

```diff
@@ -33,7 +33,7 @@
         "felixfbecker/language-server-protocol": "^1.5.2",
         "fidry/cpu-core-counter": "^0.4.1 || ^0.5.1 || ^1.0.0",
         "netresearch/jsonmapper": "^1.0 || ^2.0 || ^3.0 || ^4.0",
-        "nikic/php-parser": "^4.16",
+        "nikic/php-parser": "^5.1",
```

Comment:
Should this look like `^4.16|^5.1` as it looks like you are handling both versions through BCHelper?

Reply:
Indeed, updated